src/test/kotlin/stringCalculator/CalculatorTest.kt

```diff
@@ -0,0 +1,39 @@
+package stringCalculator
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+internal class CalculatorTest {
+
+    private val calculator = Calculator()
+
+    private fun initialize() {
+        calculator.input = ""
+    }
```

Comment:
```suggestion
    private lateinit var calculator: Calculator
    
    @BeforeEach
    fun setup() {
        calculator = Calculator()
    }
```
`@BeforeEach` 를 활용하면 매번 테스트 함수가 동작될 때마다 초기화가 되요. (이하 관련 내용 동일)

Reply:
계산기를 생각해봤을 때 계산기 한 대를 두고 reset 하면서 계산하는 게 자연스러운 흐름인 것 같아서, 🧐
계산기 instance 를 한 번 만들고, calculate 할 때 값을 입력한다 - 이렇게 구현을 했습니다.
그래서 AfterEach에 초기화 버튼을 누른다는 의미에서 input="" initializer를 구현해서 넣었습니다.

지금은 말씀해주신대로 input에 의존성이 생기니까 초기화 하는 initialize() 매서드를 하나 만들어서 호출하는 건 어떨까요?. 🙏 @BeokBeok

Reply:
우선은 위에서 리뷰해주신 내용이랑 합해서 input은 외부에서 변경이 불가능하도록 수정했습니다.